nuclei/utils.py

```diff
@@ -277,57 +273,3 @@ def message_to_python_types(
 
 def to_json(data: str) -> dict:
```

Comment:
Do we still use this function? It seems redundant with `json.loads`.

Reply:
You are right. Removed it.